src/cntk-train/src/main/scala/CommandBuilders.scala

```diff
@@ -161,14 +165,13 @@ class MPICommandBuilder(log: Logger,
 
     // Run commands below only if writing input data to HDFS
     var fileDirStr: String = ""
-    if (hdfsPath.isDefined) {
+    if (hdfsPath.isDefined && dataFormat == CNTKLearner.textDataFormat) {
```

Comment:
I think the HDFS mounting is orthogonal to text format vs. Parquet. At some point, we can add libhdfs support to the CNTK text reader for example.